spec/core/ics-003-connection-semantics/README.md

```diff
@@ -173,21 +180,25 @@ function verifyChannelState(
   proof: CommitmentProof,
   portIdentifier: Identifier,
   channelIdentifier: Identifier,
-  channelEnd: ChannelEnd) {
-    client = queryClient(connection.clientIdentifier)
-    return client.verifyChannelState(connection, height, connection.counterpartyPrefix, proof, portIdentifier, channelIdentifier, channelEnd)
+  channelEnd: ChannelEnd
+) {
+  clientState = queryClientState(connection.clientIdentifier)
+  path = applyPrefix(connection.counterpartyPrefix, channelPath(portIdentifier, channelIdentifier))
+  return verifyMembership(clientState, height, 0, 0, proof, path, channelEnd)
 }
 
-function verifyPacketData(
+function verifyPacketCommitment(
```

Comment:
I renamed this function to match ibc-go. Also, I think this name is more descriptive, since we are not verifying the actual packet data.